testing/scenario_app/bin/utils/logs.dart

```diff
@@ -26,6 +27,14 @@ void log(String msg) {
   stdout.writeln('$_gray$msg$_reset');
 }
 
+void logImportant(String msg) {
+  stdout.writeln(msg);
+}
+
+void logWarning(String msg) {
+  stderr.writeln('$_yellow$msg$_reset');
```

Comment:
nit: a function that takes in a color would be nice so not everyone has to remember to reset the color.  `colorLog(stderr, _yellow, "hello")`.

Reply:
Groovy! Done.